src/sonic-config-engine/tests/test_j2files.py

```diff
@@ -70,12 +71,17 @@ def test_dhcp_relay(self):
     def test_lldp(self):
         lldpd_conf_template = os.path.join(self.test_dir, '..', '..', '..', 'dockers', 'docker-lldp', 'lldpd.conf.j2')
 
-        # Test generation of lldpd.conf if management interface IPv4 exist
+        # Test generation of lldpd.conf if IPv4 and IPv6 management interfaces exist
         argument = '-m ' + self.t0_minigraph + ' -p ' + self.t0_port_config + ' -t ' + lldpd_conf_template + ' > ' + self.output_file
         self.run_script(argument)
         self.assertTrue(filecmp.cmp(os.path.join(self.test_dir, 'sample_output', utils.PYvX_DIR, 'lldp_conf', 'lldpd-ipv4-iface.conf'), self.output_file))
 
-        # Test generation of lldpd.conf if Management interface IPv4 doesn't exist (IPv6 only)
+        # Test generation of lldpd.conf if management interface IPv4 only exist
+        argument = '-m ' + self.t0_minigraph_ipv4_mgmt_iface + ' -p ' + self.t0_port_config + ' -t ' + lldpd_conf_template + ' > ' + self.output_file
```

Comment:
instead of using minigraph directly, i think you can use your own sample json file, it could make the test easier to read and improve.

you can use example here. construct your own json file and generate the template.

 def test_ndppd_conf(self):

Reply:
Thank you, fixed.